internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -243,23 +252,21 @@ func (r *ClusterDeploymentsReconciler) updateIfNeeded(ctx context.Context, clust
 	return update, reply, err
 }
 
-func (r *ClusterDeploymentsReconciler) notifyPullSecretUpdate(ctx context.Context, isPullSecretUpdate bool,
+func (r *ClusterDeploymentsReconciler) notifyInstallEnvUpdate(ctx context.Context, notifyInstallEnv bool,
```

Comment:
we don't support multiple InstallEnvs, and you already have the install env from `getInstallEnvByClusterDeployment` so no need to search for it

Reply:
Done